service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -32,6 +32,8 @@ oc project istio-system
 $ oc get servicemeshcontrolplanes.v1.maistra.io <smcp_name> -o yaml > <smcp_name>.v1.yaml
 ----
 +
+. Check your {ProductName} 1.1 `ServiceMeshControlPlane` resource configuration to make sure it's valid. If it's invalid, or parts that are invaild, you must fix any invalid configuration items before continuing. See xref:../../service_mesh/v2x/customizing-installation-ossm.html#ossm-updating-smcp_customizing-installation-ossm-v2x[Updating the `ServiceMeshControlPlane`].
```

Comment:
```suggestion
. Check your {ProductName} 1.1 `ServiceMeshControlPlane` resource configuration to make sure it's valid. If it's invalid, or parts that are invalid, you must fix any invalid configuration items before continuing. See xref:../../service_mesh/v2x/customizing-installation-ossm.html#ossm-updating-smcp_customizing-installation-ossm-v2x[Updating the `ServiceMeshControlPlane`].
```

Reply:
I would just remove the whole "parts" bit.

Reply:
I made this change in a different commit.